rllib/core/rl_module/marl_module.py

```diff
@@ -403,12 +402,24 @@ class MultiAgentRLModuleSpec:
         module_specs: The module specs for each individual module. It can be either a
             SingleAgentRLModuleSpec used for all module_ids or a dictionary mapping
             from module IDs to SingleAgentRLModuleSpecs for each individual module.
+        load_state_path: The path to the module state to load from. NOTE: This must be
+            an absolute path. NOTE: If the load_state_path of this spec is set, and
+            the load_state_path of one of the SingleAgentRLModuleSpecs' is also set,
+            the weights of that RL Module will be loaded from the path specified in
+            the SingleAgentRLModuleSpec. This is useful if you want to load the weights
+            of a MARL module and also manually load the weights of some of the RL
+            modules within that MARL module from other checkpoints.
+        load_state_ip_addr: The ip address of the node that the checkpoint is on. This
```

Comment:
which node is local node?
have a feeling we are using config object for experiment state passing?
this shouldn't belong to config, for cases like fault tolerance and restore for example.